pages/api/account/helpers.ts

```diff
@@ -0,0 +1,77 @@
+import sierraClient from "../../../src/server/sierraClient"
+
+export async function updatePin(
+  patronId: string,
+  barcode: string,
+  oldPin: string,
+  newPin: string
+) {
+  try {
+    const client = await sierraClient()
+    await client.post("patrons/validate", {
+      barcode: barcode,
+      pin: oldPin,
+    })
+    await client.put(`patrons/${patronId}`, { pin: newPin })
+    return { status: 200, message: `Pin updated to ${newPin}` }
+  } catch (error) {
+    return {
+      status: error.response.status,
+      message: error.response.data.message || error.response.data.description,
+    }
+  }
+}
+
+export async function updateSettings(patronId: string, patronData: any) {
```

Comment:
I think this could use a more descriptive title (updatePatronSettings / updatePatronData?) and possibly a comment.